operator-common/src/main/java/io/strimzi/operator/cluster/model/ClientsCa.java

```diff
@@ -37,6 +40,20 @@ public static Secret adapt060ClientsCaSecret(Secret clientsCaKey) {
         return clientsCaKey;
     }
 
+    public void initCaSecrets(Secret secret) {
+        this.brokersSecret = secret;
+    }
```

Comment:
Its `initCaSecrets` in plural. So, should it mirror the ClusterCA method and take a list of secrets despite currently needing only one secret from the list? The fact that you use the same name but different parameter and behaviour seems weird. I would either rename ti to match the actual purpose (e.g. `initBrokerSecret` or `setBrokerSecret`) or mirror the behaviour of the ClusterCa class and just pick the broker secret.

Reply:
As discussed offline, It's kind of problematic doing the same as ClusterCa because we would need the cluster name to filter the corresponding broker Secret name. It would be ok if ClientsCa would be used in the CO only but it's used in the UO which can run as standalone as well so we cannot rely on the `strimzi.io/cluster` label selector for getting such an info. I will go to just rename the method in a better way.